Templates/MonoGame.Templates.CSharp/content/MonoGame.2D.StartKit/___SafeGameName___.Core/Content/___SafeGameName___.contentproj

```diff
@@ -0,0 +1,320 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This is an old XNA content project?

Reply:
@dellis1972 I copied it from platformer. I'll see what the project templates generate.

Reply:
We should not be including the old XNA projects. They are not needed. Nothing can build them now afaik.